app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
```

Comment:
I won't hold the PR up either way, just expressing my opinion on how this can lead to developers wasting their time (or it might just be me wasting my time :))